### ISSUE TITLE
OpenSearch, ElasticSearch module

### ISSUE THREAD

**Issue by sebastien-rosset** [ansible-collections/amazon.aws#609]
### Summary

It would be useful to have a `ec2_opensearch` module to deploy opensearch/elasticsearch clusters. Is it missing because nobody from the community submitted a PR? Or was it intentionally not added? Some other reason?

There is one module outside this repo (https://github.com/fiunchinho/ansible-aws-elasticsearch-module), but it hasn't been updated in the last 3 years.

### Issue Type

Feature Idea

### Component Name

ec2_opensearch

### Additional Information

```yaml (paste below)

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by alinabuzachis** [comment]
@sebastien-rosset Thank you for this idea. Would you be willing to work on this module and submit a PR?

**Comment by sebastien-rosset** [comment]
> @sebastien-rosset Thank you for this idea. Would you be willing to work on this module and submit a PR?
Yes

**Comment by alinabuzachis** [comment]
@sebastien-rosset Awesome! Please feel free to work on it! :-)

**Comment by jillr** [comment]
I agree with @alinabuzachis, the new module would go into community.aws.  It should be named `opensearch` rather than `ec2_opensearch` as this service does not use the EC2 API.
